Formula/go.rb

```diff
@@ -3,16 +3,20 @@ class Go < Formula
   homepage "https://golang.org"
 
   stable do
-    url "https://storage.googleapis.com/golang/go1.7.5.src.tar.gz"
-    mirror "https://fossies.org/linux/misc/go1.7.5.src.tar.gz"
-    version "1.7.5"
-    sha256 "4e834513a2079f8cbbd357502cccaac9507fd00a1efe672375798858ff291815"
+    # Version bumps of this formula require formulae that depend on Go at
+    # runtime to be rebuilt in order to pass their tests. This can be done by
+    # bumping those formula's revision.
```

Comment:
Thanks for the suggestions. I don't want to bikeshed this longer than we need to, so I'm alright with deferring to @ilovezfs's judgement, if he's OK with contributing here.

I've ticked the 'Allow edits from maintainers.' box, so anyone from Homebrew core is welcome to touch this.